src/neuroconv/datainterfaces/behavior/medpc/medpcdatainterface.py

```diff
@@ -0,0 +1,254 @@
+            A dictionary mapping the names of the variables to the synchronized timestamps for data in this interface.
+        """
+        self.timestamps_dict = aligned_timestamps_dict
+
+    def set_aligned_starting_time(self, aligned_starting_time: float, medpc_name_to_info_dict: dict) -> None:
+        """
+        Align the starting time for this interface relative to the common session start time.
+
+        Must be in units seconds relative to the common 'session_start_time'.
+
+        Parameters
+        ----------
+        aligned_starting_time : float
+            The starting time for all temporal data in this interface.
+        medpc_name_to_info_dict : dict
+            A dictionary mapping the names of the desired variables in the MedPC file
+            to an info dictionary with the names of the variables in the metadata and whether or not they are arrays.
+            ex. {"A": {"name": "left_nose_poke_times", "is_array": True}}
+        """
+        original_timestamps_dict = self.get_original_timestamps(medpc_name_to_info_dict=medpc_name_to_info_dict)
+        aligned_timestamps_dict = {}
+        for name, original_timestamps in original_timestamps_dict.items():
+            aligned_timestamps_dict[name] = original_timestamps + aligned_starting_time
+        self.set_aligned_timestamps(aligned_timestamps_dict=aligned_timestamps_dict)
+
+    def add_to_nwbfile(
+        self,
+        nwbfile: NWBFile,
+        metadata: dict,
+        medpc_name_to_info_dict: Optional[dict] = None,
+        module_name: Optional[str] = None,
+        module_description: Optional[str] = None,
+        events: Optional[list[dict]] = None,
+        interval_series: Optional[list[dict]] = None,
+    ) -> None:
+        ndx_events = get_package(package_name="ndx_events", installation_instructions="pip install ndx-events")
+        medpc_name_to_info_dict = metadata["MedPC"].get("medpc_name_to_info_dict", medpc_name_to_info_dict)
+        info_name_to_medpc_name = {
+            info_dict["name"]: medpc_name for medpc_name, info_dict in medpc_name_to_info_dict.items()
+        }
+        for name in self.source_data["aligned_timestamp_names"]:
+            medpc_name = info_name_to_medpc_name[name]
+            medpc_name_to_info_dict.pop(medpc_name)
+        session_dict = read_medpc_file(
+            file_path=self.source_data["file_path"],
+            medpc_name_to_info_dict=medpc_name_to_info_dict,
+            session_conditions=self.source_data["session_conditions"],
+            start_variable=self.source_data["start_variable"],
+        )
+        aligned_timestamps_dict = self.get_timestamps()
+        for name, aligned_timestamps in aligned_timestamps_dict.items():
+            session_dict[name] = aligned_timestamps
+
+        # Add behavior data to nwbfile
+        module_name = metadata["MedPC"].get("module_name", module_name)
+        module_description = metadata["MedPC"].get("module_description", module_description)
```

Comment:
It looks like, from the argument setting above, that this call to `.get` would by default return 'None' if the metadata field is not specified, correct?

I would remove the duplication of this information (module name and description) by choosing it to be specified fully in either `metadata` or as `conversion_options` (not both)

I would also set a static default